gsheets-workflow-api/lib/gs_combined/schemas.py

```diff
@@ -143,18 +153,21 @@ class GsSurveyResultsData:
             "weighted_by": "Weighted by",
         },
         "imported_igno_questions": {
+            # First group - Igno Index
             "igno_index_question_id": "ID\ncombo",
             "igno_index_world_views_survey_batch_number": "WV#",
             "igno_index_question": "Question ",
             "igno_index_question_correct_answer": "Correct Answer",
             "igno_index_question_answer_options": "Answer options",
             "igno_index_question_very_wrong_answer": "Very Wrong Answer - filled out only if it can't be derived numerically",
+            # Second group - Foreign Country
             "foreign_country_igno_question_id": "ID combo",
-            "foreign_country_country_views_survey_batch_number": "CV#",
+            "country_views_survey_batch_number": "CV#",  # Fixed attribute name to match usage
```

Comment:
This change was suggested by AI, I think it make sense because in other modules, we only refer to "country_views_survey_batch_number", not "foreign_country_country_views_survey_batch_number"